openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx

```diff
@@ -263,16 +272,17 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
         />
       </Form.Item>
       <Form.Item
-        label="Effect:"
+        label={`${t('label.effect')}:`}
         name="ruleEffect"
         rules={[
           {
             required: true,
+            message: t('label.field-required', { field: 'Effect' }),
```

Comment:
```suggestion
            message: t('label.field-required', { field: t('label.effect') }),
```

Update same for others as well

Reply:
done.